Releases.md

```diff
@@ -23,6 +24,12 @@ See [DirectML version history on MSDN](https://docs.microsoft.com/windows/win32/
 | [1.1.0](#directml-110)   | [DML_FEATURE_LEVEL_2_0](https://learn.microsoft.com/windows/ai/directml/dml-feature-level-history#dml_feature_level_2_0) | Windows 10 May 2020 Update, Version 2004 (Build 10.0.19041, 20H1) | -                                                                                           |
 | [1.0.0](#directml-100)   | [DML_FEATURE_LEVEL_1_0](https://learn.microsoft.com/windows/ai/directml/dml-feature-level-history#dml_feature_level_1_0) | Windows 10 May 2019 Update, Version 1903 (Build 10.0.18362; 19H1) | -                                                                                           |
 
+# DirectML 1.12.0
+
+- Introduced DML_FEATURE_LEVEL 6.1
+  - Added DML_OPERATOR_MULTIHEAD_ATTENTION.
+  - Added DML_OPERATOR_ACTIVATION_SOFTMAX/1 to the list of fusable activations for DML_OPERATOR_GEMM.
```

Comment:
it should be `DML_OPERATOR_ACTIVATION_SOFTMAX` and `DML_OPERATOR_ACTIVATION_SOFTMAX1`